@types/index.d.ts

```diff
@@ -6,6 +6,8 @@ declare module '@rebass/preset';
 
 // read about the audio features here: https://meyda.js.org/audio-features
 interface Sound {
+    _id?: string;
+    filename?: string;
     chroma?: number[];
     filename: string;
```

Comment:
We shouldn't have both L10 and L12.

Reply:
@julesyoungberg Nice catch, I completely missed that it was already defined 😳.

I agree with the purple maybe not being the best color for the UI (I just used the rebass accent colors).
We can easily change the rebass theme later.